src/pages/play.astro

```diff
@@ -1,48 +1,18 @@
 ---
 import i18next, { t, changeLanguage } from 'i18next'
-import { HeadHrefLangs } from 'astro-i18next/components'
+import Base from "../layouts/Base.astro";
 
-const title = `Playground | ${t('site.title')}`
-const description = t('site.description')
-const image = '/img/wm-social.png'
-const canonicalURL = new URL(Astro.url.pathname, Astro.site)
+changeLanguage("en");
 
-changeLanguage('en')
+const title = `Playground | ${t("site.title")}`;
+const description = t("site.description");
 ---
 
-<html lang={i18next.language} data-theme='light'>
-  <head>
-    <meta charset='utf-8' />
-    <meta name='viewport' content='width=device-width,initial-scale=1' />
-    <link rel='icon' type='image/svg+xml' href='/favicon.svg' />
-    <meta name='generator' content={Astro.generator} />
-
-    <link rel='canonical' href={canonicalURL} />
-
-    <title>{title}</title>
-    <meta name='title' content={title} />
-    <meta name='description' content={description} />
-
-    <meta property='og:type' content='website' />
-    <meta property='og:url' content={Astro.url} />
-    <meta property='og:title' content={title} />
-    <meta property='og:description' content={description} />
-    <meta property='og:image' content={new URL(image, Astro.url)} />
-
-    <meta property='twitter:card' content='summary_large_image' />
-    <meta property='twitter:url' content={Astro.url} />
-    <meta property='twitter:title' content={title} />
-    <meta property='twitter:description' content={description} />
-    <meta property='twitter:image' content={new URL(image, Astro.url)} />
-
-    <HeadHrefLangs />
-  </head>
-  <body>
-    <main>
-      <div class='header'>
-        <img src='/img/wm-logo--full-animated.svg' width='400' height='52' />
-        <h1>Playground</h1>
-      </div>
+<Base title={title} description={description}>
+  <section class="section">
+    <div class="contentWrapperMd">
```

Comment:
We want to utilize all the available width. Can we remove the `contentWrapperMd` our update so it doesn't have a max width and an inline padding of 2rem.

Reply:
ok